src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs

```diff
@@ -16,17 +16,12 @@ namespace AWS.Deploy.CLI.Utilities
     public class CommandLineWrapper : ICommandLineWrapper
     {
         private readonly IOrchestratorInteractiveService _interactiveService;
-        private readonly AWSCredentials _awsCredentials;
-        private readonly string _awsRegion;
+        private Action<ProcessStartInfo> _processStartInfoAction;
```

Comment:
nit:  This _feels_ like it would be better as a collection of actions, ie `List<Action<ProcessStartInfo>>` to better align with a "plugin" pattern.  Basically, you could register multiple plugins rather then the last to call `ConfigureProcess` wins.

But not a big deal if you want to play the YAGNI card.